examples/restaurant_visits/run_without_frameworks_utility_analysis.py

```diff
@@ -23,6 +23,8 @@
 from absl import flags
 import pipeline_dp
 import pandas as pd
+
+import utility_analysis_new.utility_analysis
 from utility_analysis_new.dp_engine import UtilityAnalysisEngine
```

Comment:
Does the guideline about not importing classes not apply to main classes?

Reply:
This is not production code, just an example, it's okeish :)